governance/polls/Offboard USDT Collateral Types - September 27, 2021.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Offboard USDT Collateral Types - September 27, 2021
+summary: Signal your support or opposition to Offboard USDT Collateral Types.
+discussion_link: https://forum.makerdao.com/t/signal-request-offboard-usdt-collateral-types/10223
+vote_type: Plurality Voting
+categories:
+   - Collateral
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2021-09-27T16:00:00
+end_date: 2021-09-30T16:00:00
+---
+
+# Poll: Offboard USDT Collateral Types - September 27, 2021
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the MakerDAO Community. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, September 27 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes: 
+* Lower the USDT-A [Liquidation Penalty (chop)](https://makerdao.world/en/learn/governance/param-liquidation-penalty/) to 0%.
+* Use the lerp module to increase the USDT-A [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio/) to 300% to trigger liquidation of all remaining vaults.
```

Comment:
Are we lerping? No mention of it in the signal request - I thought the liquidation ratio would increase instantly rather than gradually